modules/spo-recording-profiles.adoc

```diff
@@ -99,7 +101,7 @@ $ oc -n openshift-security-profiles logs --since=1m --selector name=spod -c log-
 .Example output
 [source,terminal,subs="attributes+"]
 ----
-I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "container"="redis" "namespace"="my-namespace" "node"="ip-10-0-189-53.us-east-2.compute.internal" "perm"="name_bind" "pod"="my-pod" "profile"="test-recording_redis_6kmrb_1684331729" "scontext"="system_u:system_r:selinuxrecording.process:s0:c4,c27" "tclass"="tcp_socket" "tcontext"="system_u:object_r:redis_port_t:s0" "timestamp"="1684331735.105:273965" "type"="{type}"
+I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server""namespace"="my-namespace" "node"="ip-10-0-189-53.us-east-2.compute.internal" "pid"=656802 "pod"="my-pod" "syscallID"=0 "syscallName"="read" "timestamp"="1684331735.105:273965" "type"="seccomp"
```

Comment:
Is it possible to split `Managing seccomp profiles ` and `Managing SELinux profiles` into two files?
For selinx: the output should be:
`I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "container"="redis" "namespace"="my-namespace" "node"="ip-10-0-189-53.us-east-2.compute.internal" "perm"="name_bind" "pod"="my-pod" "profile"="test-recording_redis_6kmrb_1684331729" "scontext"="system_u:system_r:selinuxrecording.process:s0:c4,c27" "tclass"="tcp_socket" "tcontext"="system_u:object_r:redis_port_t:s0" "timestamp"="1684331735.105:273965" "type"="selinux"`
For seccomp, the output should be:
`I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server""namespace"="my-namespace" "node"="ip-10-0-189-53.us-east-2.compute.internal" "pid"=656802 "pod"="my-pod" "syscallID"=0 "syscallName"="read" "timestamp"="1684331735.105:273965" "type"="seccomp"`

Reply:
Oh! Yes!